EIPS/eip-4788.md

```diff
@@ -80,7 +80,9 @@ When verifying a block, execution clients **MUST** ensure the root value in the
 #### Block processing
 
 At the start of processing any execution block where `block.timestamp >= FORK_TIMESTAMP` (i.e. before processing any transactions),
-write the parent beacon root provided in the block header into the storage of the contract at `HISTORY_STORAGE_ADDRESS`.
+
+- Write the parent beacon root provided in the block header into the storage of the contract at `HISTORY_STORAGE_ADDRESS`.
+- If the `HISTORY_STORAGE_ADDRESS` has `nonce==0`, set `nonce` to `1`. This is to prevent touch-delete rules from deleting the account from the state.
```

Comment:
can't we just set the account to some clear state without conditions, so we can say
- Set the HISTORY_STORAGE_ADDRESS's nonce to 1 and balance to 0